app/templates/components/ecosystem-icon.hbs

```diff
@@ -1,4 +1,4 @@
 <div class="ecosystem-icon">
   <svg class="background" xmlns="http://www.w3.org/2000/svg" width="133" height="130"><defs/><path fill={{@bg}} fill-rule="nonzero" d="M.91 27.535l15.047 93.176c.878 5.44 5.993 9.145 11.436 8.283l96.19-15.233c5.45-.863 9.172-5.98 8.314-11.43L117.264 9.301c-.857-5.451-5.969-9.178-11.42-8.326L9.238 16.06a10 10 0 00-8.33 11.475z"/></svg>
-  <img class="logo {{if @rotate "rotate"}}" src="/images/home/logos/{{@icon}}.svg" alt={{@icon}}>
+  <img class="logo {{if @rotate "rotate"}}" src="/images/home/logos/{{@icon}}.svg" alt={{@icon}} role="presentation" loading="lazy">
```

Comment:
The icons need their Alt text, so role presentation isn’t correct here.

Reply:
```suggestion
  <img class="logo {{if @rotate "rotate"}}" src="/images/home/logos/{{@icon}}.svg" alt={{@icon}} loading="lazy">
```